interface/patient_file/summary/labdata.php

```diff
@@ -81,6 +81,7 @@
 // ####################################################
 echo "<html><head>";
 ?> 
+<title><?php echo xlt('Lab Overview'); ?></title>
 <link rel="stylesheet" href="<?php echo $css_header;?>" type="text/css"> 
```

Comment:
Yop this does the trick. Good catch

Reply:
Thanks.

Reply:
If "lab overview" does not need to be echoed onto the form (it is already in the tab) then set the title to -hidden.

Reply:
@jonathanzhang53 i think #1331 needs this

Reply:
Cool I'll check it out.